hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -1461,15 +1461,15 @@ possible configurations would overwhelm and obscure the important.
   </property>
   <property>
     <name>dfs.client.read.shortcircuit</name>
-    <value>false</value>
+    <value></value>
```

Comment:
Will this cause he loading of configurations failing? Not sure...

Reply:
hdfs-site.xml should be taking precedence over this property (I had a similar question about dfs.domain.socket.path) and this shouldn't be an issue in real life.

However, I would also be ok coming back and just dropping these entries entirely in the future :)